### PR TITLE
First steps in the project

### PR DESCRIPTION
**Pull request by enriqueHG94**
I have created the project and the folder with the resources I am going to use, then I have created the file with which i'm going to work with PySpark, in this first step I have created the Spark session and I have read a file in CSV format and later I have shown it on the screen.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (jaBaena) [inline .idea/.gitignore:3]: I would add the .idea folder here, as they are IDE configuration files that we don't need

Comment (enriqueHG94) [reply .idea/.gitignore]: I have deleted the idea folder and added the gitignore file.

File: Main.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from pyspark.sql import SparkSession
+
+# Crea una sesión de Spark
+spark = SparkSession.builder.appName('reading CSV').getOrCreate()
+
+# Lee el archivo CSV desde la carpeta
+df = spark.read.format('csv')\
+    .option('header', 'true')\
+    .load("Resources\Loan-Data.csv")
```
[the remark this answers is not in the document]
Comment (enriqueHG94) [reply Main.py:9]: done, thanks

File: Main.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from pyspark.sql import SparkSession
+
+# Crea una sesión de Spark
```
[the remark this answers is not in the document]
Comment (enriqueHG94) [reply Main.py]: ok, i understand what you mean, i will try to put only comments when the code is more complex.

**Review by jaBaena — APPROVED** [review]
Approved, good work!

**Pull request merged**